sycl/test/reduction/reduction_nd_s1_rw.cpp

```diff
@@ -87,7 +87,7 @@ int main() {
 
   // Check with various types.
   test<float, 1, std::multiplies<float>>(1, 8, 256);
-  test<float, 1, intel::minimum<float>>(getMaximumFPValue<float>(), 8, 256);
+  test<float, 1, intel::minimum<float>>(getMaximumFPValue<float>(), 1, 16);
```

Comment:
Just double-checking that this change was deliberate.

Reply:
Yes, it was. The new 1 is the size of the work-group and 16 is the number of elements to reduce. 16 is just enough here.